angular-ngrx-scss/src/app/state/profile/profile.effects.ts

```diff
@@ -16,12 +18,54 @@ export class ProfileEffects {
       switchMap(({ username }) =>
         combineLatest([
           this.userService.getUserInfo(username),
-          this.userService.getUserOrganizations(),
+          this.userService.getUserOrganizations(username),
           this.userService.getUserRepos(username),
         ]).pipe(
-          map(([user, orgs, repos]) =>
-            fetchProfileSuccess({ data: { user, orgs, repos } }),
-          ),
+          map(([userData, orgsData, reposData]) => {
+            const user: UserState = {
+              avatar: userData.avatar_url,
+              bio: userData.bio,
+              blog: userData.blog,
+              company: userData.company,
+              email: userData.email,
+              followers: userData.followers,
+              following: userData.following,
+              location: userData.location,
+              name: userData.name,
+              twitter_username: userData.twitter_username,
```

Comment:
IMO we shouldn't keep the fields' names exactly as they are in the API response but rather convert to conventions that we use in JS/TS ie. use camelCase instead of snake_case

Reply:
The `twitter_username` was pretty easy to adjust, but a lot of the others are tied more directly to API calls. We have a ticket created already to adjust the interfaces - I'm going to mark in there to also try to adjust these a bit more for the app-facing ones so we're consistent.